packages/playground/public/info/minting.md

```diff
@@ -17,4 +17,4 @@ The TFT minting address on Stellar Chain:
 
 <br />
 
-For more questions about minting, please contact the [support team](https://threefold.io/support/) or click on the chat icon.
+For more questions about minting, please contact the [support team](https://threefoldfaq.crisp.help/en/) or click on the chat icon.
```

Comment:
@AhmedHanafy725 do we use crisp here or update it to another link?

Reply:
please use threefold.io instead

Reply:
unblock it with the crisp url, the marketing team will be restoring a support page soonish